tests/integration/study.js

```diff
@@ -46,28 +46,5 @@ describe("Study Template integration test example", function () {
     await this.driver.setContext(firefox.Context.CHROME);
     await findAndAct(this.driver, By.css(`[label="Add"]`), e => e.click());
     await findAndAct(this.driver, By.css(`[label="Okay, Got It"]`), e => e.click());
```

Comment:
nit: can you change `"successfully opens the study template options page on installation"` to `"successfully installs the study"`? Unfortunately GitHub does not allow me to leave the comment there :(

Reply:
good catch, will change that.